static/src/js/actions/__tests__/urlQuery.test.js

```diff
@@ -10,6 +10,121 @@ import * as urlQuery from '../urlQuery'
 
 const mockStore = configureMockStore([thunk])
 
+jest.mock('../../../../../portals/availablePortals.json', () => ({
+  airmoss: {
```

Comment:
If you need to mock values because we are testing for them, you should use fake values, not the real portals. Using real values can lead to confusion later if the portals change and the values are no longer synced. I realize this is not how we did it with the availablePortalsMock.json, but we should improve and do it better than we used to

Reply:
I agree! Good callout I did need to use `virtual mocks` in one place since the png file that jest tries to get does not actually exist on the disk